R/content.R

```diff
@@ -747,6 +747,31 @@ terminate_jobs <- function(content, keys = NULL) {
   res_df
 }
 
+#' Get Job Log
+#'
+#' Get the log output for a job. Requires Connect 2022.10.0 or newer.
+#'
+#' @param content A Content object, as returned by `content_item()`
+#' @param key A job key, which you can find using `get_jobs()`.
+#'
+#' @return A data frame with the requested log. Each row represents an entry.
+#'
+#' - `source`: `stdout` or `stderr`
+#' - `timestamp`: The time of the entry.
+#' - `data`: The logged text.
+#'
+#' @family job functions
+#' @family content functions
+#' @export
```

Comment:
Adding literal `@examples` here is complicated (though could be done with httptest!) would you mind writing one in the comment here (or on the main PR), what is the code someone would write to use this?

Reply:
Yep, this was on my list of things to do, thanks for the reminder.

Reply:
Noticing that the other jobs functions also lack examples (which is why I didn't remember to add one when I was writing this documentation). Going to add those as well.